manifests/weave/boarnet/bns/genesis.json

```diff
@@ -0,0 +1,458 @@
+      },
+      {
+        "address": "28691FFE125E01B306614121EAC07FEA14917781",
+        "//name": "Harry",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "AD102CF85A366D726BBBCA54DBD3818B6278A66A",
+        "//name": "Ingrid",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "0EDAA1225975A3A64BC7216F9429A3EA34A757D9",
+        "//name": "Jonny",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "BA1DBBA7C4A04F39A98FA5B4DBA8BC6981D7D40E",
+        "//name": "Cloe",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "cond:gov/rule/0000000000000002",
+        "//name": "reward fund",
+        "coins": [
+          {
+            "whole": 1,
+            "ticker": "IOV"
+          }
+        ]
+      },
+      {
+        "address": "0E95C039EF14EE329D0E09D84F909CF9EB5EF472",
+        "//name": "faucet",
+        "coins": [
+          {
+            "whole": 999999999,
+            "ticker": "IOV"
+          }
+        ]
+      },
+      {
+        "address": "B1DE2F431945B8D7CE20DDD5F43FDA9609E9D7B8",
+        "//name": "loadtester",
+        "coins": [
+          {
+            "whole": 999999999,
+            "ticker": "IOV"
+          }
+        ]
+      }
+    ],
+    "currencies": [
+      {
+        "ticker": "IOV",
+        "name": "Main token of this chain"
+      }
+    ],
+    "multisig": [
+      {
+        "activation_threshold": 1,
+        "admin_threshold": 1,
+        "participants": [
+          {
+            "signature": "BA1DBBA7C4A04F39A98FA5B4DBA8BC6981D7D40E",
+            "//name": "Cloe",
+            "weight": 1
+          }
+        ]
+      }
+    ],
+    "update_validators": {
+      "addresses": [
+        "cond:gov/rule/0000000000000004"
+      ]
+    },
+    "distribution": [
+      {
+        "//name": "unused - owned by economic committee",
+        "admin": "cond:gov/rule/0000000000000002",
+        "destinations": [
+          {
+            "address": "0E95C039EF14EE329D0E09D84F909CF9EB5EF472",
+            "weight": 1
+          }
+        ]
+      }
+    ],
+
+    "escrow": [
+      {
+        "//name": "guarantee fund",
+        "amount": [
+          {
+            "ticker": "IOV",
+            "whole": 1000000
+          }
+        ],
+        "arbiter": "cond:gov/rule/0000000000000002",
+        "destination": "cond:gov/rule/0000000000000002",
+        "//destination": "reward fund - cash account",
+        "source": "0000000000000000000000000000000000000000",
+        "timeout": "2999-12-31T00:00:00Z"
+      },
+      {
+        "//name": "vesting first tranche",
+        "amount": [
+          {
+            "ticker": "IOV",
+            "whole": 1234567
+          }
+        ],
+        "arbiter": "0000000000000000000000000000000000000000",
+        "destination": "04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17",
```

Comment:
Account `04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17` does not have any tokens to access the escrow. Can you add a wallet, please?

Reply:
Are the credentials for `04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17` shared?

Reply:
No, they're not shared.  It's my testnet account.

Reply:
> Account `04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17` does not have any tokens to access the escrow. Can you add a wallet, please?

```
[iov@debt boarnet]$ env | fgrep BNS
BNSCLI_TM_ADDR=https://rpc.boarnet.iov.one:443
[iov@debt boarnet]$ bnscli query -path=/wallets  -data=04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17 | jq
[
  {
    "Key": "636173683a04c3db7cccacf58eefcc296ff7ad0f6db7c2fa17",
    "Value": {
      "metadata": {
        "schema": 1
      },
      "coins": [
        {
          "whole": 1,
          "ticker": "IOV"
        }
      ]
    }
  }
]
```